array-manipulation-2/src/take.js

```diff
@@ -0,0 +1,39 @@
+/* exported take */
+
+/*
+  - Make new array variable
+  - Loop through array with condition i less than input count
+    - If length of array is strictly equal to 0,
+      - then return empty array
+    - Push current index of array into new array variable
+  - Return new array variable outside loop
+*/
+
+function take(array, count) {
+  var newArray = [];
+  for (var i = 0; i < count; i++) {
+    if (array.length === 0) {
+      return [];
+    }
```

Comment:
Same here. No need to even start the loop if the array is empty. Alternatively, you could make your loop's condition expression more robust.

```js
for (var i = 0; i < count && i < array.length; i++) {
```